### PR TITLE
Fix quantization dependency

### PR DESCRIPTION
**Pull request by skyw**
There is only one python version of prebuild wheel of TensorRT's pytorch-quantization on nvidia pyindex, which causes install with all other version of python to fail with a confusion error message.

Removed requirement_quantization.txt. Instead, install from source in Dockerfile. 

Also added error message if running quantization scripts without pytorch-quantization.

@okuchaiev

### CODE REVIEW

File: examples/asr/speech_to_text_quant_infer.py [hunk 1]
```diff
@@ -21,13 +21,21 @@
 from pprint import pprint
 
 import torch
-from pytorch_quantization import nn as quant_nn
 from pytorch_quantization import quant_modules
```
Comment (titu1994) [inline examples/asr/speech_to_text_quant_infer.py:24]: There's an additional import here

Comment (skyw) [reply examples/asr/speech_to_text_quant_infer.py:24]: black moved things around, the check was on the first import. will move all the pytorch_quantization import under the try clause.

File: Dockerfile [hunk 2]
```diff
@@ -73,6 +73,13 @@ WORKDIR /tmp/nemo
 COPY requirements .
 RUN for f in $(ls requirements/*.txt); do pip install --disable-pip-version-check --no-cache-dir -r $f; done
 
+# install quantization support
+RUN git clone https://github.com/NVIDIA/TensorRT.git && \
```
Comment (okuchaiev) [inline Dockerfile:77]: @skyw do you want to take a specific commit here. Or you are confident that TOT will be fine?

Comment (skyw) [reply Dockerfile:77]: TOT should be fine. It is only updated once a month, together with other DLFW update.

**Review by okuchaiev — APPROVED** [review]

**Pull request merged**